docs/serverless/ingest/auto-import.mdx

```diff
@@ -0,0 +1,85 @@
+</DocCallOut>
+
+
+## Create a new custom integration
+
+1. In your ((elastic-sec)) deployment, click **Add integrations**.
+2. Under **Can't find an integration?** click **Create new integration**.
+
+<DocImage url="images/auto-import-create-new-integration-button.png" alt="The Integrations page with the Create new integration button highlighted" />
+
+3. Click **Create integration**.
+4. Select an <DocLink slug="/serverless/security/connect-to-bedrock" text="Amazon Bedrock connector"/>. 
+5. Define how your new integration will appear on the integrations page by providing a **Title**, **Description**, and **Logo**.  Click **Next**.
+6. Define your integration's package name, which will prefix the imported event fields. 
+7. Define your **Data stream title**, **Data stream description**, and **Data stream name**. These fields will appear on the integration's configuration page to help identify the data stream it writes to.
+8. Select your [**Data collection method**](https://www.elastic.co/guide/en/beats/filebeat/current/configuration-filebeat-options.html). This determines how your new integration will ingest the data, for example from an S3 bucket, an HTTP endpoint, or a file stream.
+9. Upload a sample of your data in JSON format. Make sure to include all the types of events that you want the new integration to handle. 
+<DocCallOut title="Best practices for sample data"> 
+- The file extension (`.JSON` or `.NDJSON`) must match the file format.
+- Only the first ten events in the sample are analyzed; in this technical preview, additional data is truncated. 
+- Ensure each JSON or NDJSON object represents an event, and avoid deeply-nested object structures. 
+- The more variety in your sample, the more accurate the pipeline will be (for example, include 10 unique log entries instead of the same type of entry 10 times).
+- Ideally, each field name should describe what the field does.
+</DocCallOut>
+10. Click **Analyze logs**, then wait for processing to complete. This may take several minutes.
+11. After processing is complete, the pipeline's field mappings appear, including both ECS and custom fields.
+
+<DocImage url="images/auto-import-review-integration-page.png" alt="The Automatic Import Review page showing proposed field mappings" />
+
+12. (Optional) After reviewing the proposed pipeline, you can finetune it by clicking **Edit pipeline**. Refer to the [((elastic-sec)) ECS reference](https://www.elastic.co/guide/en/security/current/siem-field-reference.html) to learn more about how field mappings should look. When you're satisfied with your changes, click **Save**. 
+
+<DocImage url="images/auto-import-edit-pipeline.gif" alt="A gif showing the user clicking the edit pipeline button and viewing the ingest pipeline flyout" />
+
+<DocCallOut title="Tip">
+You can use the <DocLink slug="/serverless/security/data-quality-dash" text="Data Quality dashboard"/> to check the health of your data ingest pipelines and field mappings.
+</DocCallOut>
```

Comment:
Are we expecting the user to do this _right here, right now_ in the middle of this procedure? It seems like it'd be disruptive. Either relocate the tip outside the procedure, or rewrite to make it clear when exactly they should check out the DQ dashboard.